### ISSUE TITLE
Error while fetching file: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:720)>.

### ISSUE THREAD

**Issue by rhabacker** [domlysz/BlenderGIS#184]
With blender 2.79b I get the following error on activating BlenderGIS addon:

```imageio: 'libfreeimage-3.16.0-linux64.so' was not found on your computer; downloading it now.
Error while fetching file: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:720)>.
Error while fetching file: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:720)>.
Error while fetching file: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:720)>.
Can't install imageio lib Unable to download 'libfreeimage-3.16.0-linux64.so'. Perhaps there is a no internet connection? If there is, please report this problem.
```

**Comment by rhabacker** [comment]
A workaround I found out is to download the corresponding file with wget or curl and save it in the BlenderGIS subfolder core/lib/imageio/resources/freeimage.

**Comment by domlysz** [comment]
same as https://github.com/domlysz/BlenderGIS/issues/165

**Comment by rhabacker** [comment]
Thank you very much for that hint:
I understand that this problem can be fixed for Blender 2.79 by applying commit https://github.com/domlysz/BlenderGIS/commit/662a668ad22e05e7455d7a70482a19c32c05b558 to the Blender279 branch ?
